<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Left_pushButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>350</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
</string>
    </property>
    <property name="text">
     <string>Bit Shift Left - Encode</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Con_textEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;
color: rgb(85, 255, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Con_Clear_pushButton_2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>280</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
</string>
    </property>
    <property name="text">
     <string>Clear All Screen</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Con_textEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;
color: rgb(85, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 11pt &quot;Arial&quot;;
color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string> ASCII Shift Encoder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 11pt &quot;Arial&quot;;
color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string> ASCII Shift Decoder</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Con_textEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;
color: rgb(85, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 11pt &quot;Arial&quot;;
color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string> Input String For The Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Left_pushButton_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>420</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
</string>
    </property>
    <property name="text">
     <string>Bit Shift Left - Decode</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>70</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Hex</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1314</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
